src/main/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepository.java

```diff
@@ -25,12 +24,12 @@
 import javax.persistence.PersistenceContext;
 import javax.persistence.PersistenceException;
 import javax.persistence.Query;
-
-import org.hibernate.StaleObjectStateException;
-import org.hibernate.exception.ConstraintViolationException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Qualifier;
+import java.time.LocalDateTime;
```

Comment:
imports seems to have changed in order. could we make sure we're using the standard code format activated to be applied on auto-save action?

Reply:
Imports are fine. Ordered by the formatter.